packages/sources/src/components/accordion/accordion.models.ts

```diff
@@ -0,0 +1,25 @@
+export enum AccordionVariant {
```

Comment:
Deze enum wordt niet uitgebuit: Ja, je hebt 'm als type bij `interface Accordion.variant`, maar zoals we gisteren hadden ontworpen is de variant "Default" niet `dso-accordion` maar "niets" (vandaar undefined).

Ik denk dat je de constructie met enum's voor nu maar moet laten.

Reply:
Prima, ik haal hem weg